discovery-infra/monitoring/process.py

```diff
@@ -0,0 +1,94 @@
+import re
+import json
+import logging
+import collections
+
+REMOVED_FIELDS = [
+    "cluster.image_info_ssh_public_key",
+    "cluster.ssh_public_key",
+    "cluster.image_info.ssh_public_key"
+    "cluster.connectivity_majority_groups",
+    "cluster.controller_logs_collected_at",
+    
+    "cluster.hosts.connectivity",
+    "cluster.hosts.images_status",
+
+    "cluster.image_info_download_url",
+    "cluster.image_info_expires_at",
+    "cluster.image_info_size_bytes",
+    "cluster.ingress_vip",
+    "link"
+]
+
+SKIPPABLE_EVENTS = [
+    "reached installation stage Writing image to disk"
+]
+
+logging.basicConfig(level=logging.WARN, format='%(levelname)-10s %(message)s')
+logger = logging.getLogger(__name__)
+logging.getLogger("__main__").setLevel(logging.INFO)
+
+
+def convert_field_to_json(converted_field):
+    try:
+        if type(converted_field) == str:
+            return json.loads(converted_field)
+    except KeyError:
```

Comment:
sure it fails only on KeyError ?

Reply:
if it fails on something else we want to get notified about it